core/src/main/java/com/linecorp/armeria/common/logging/DefaultRequestLog.java

```diff
@@ -892,16 +898,16 @@ public String toString() {
            .append(res)
            .append('}');     // 1 char
         if (children != null && children.size() > 1) {
```

Comment:
- You could define a variable upfront and reuse it everywhere you call `children.size()`:

  ```java
  final int numChildren = children != null ? children.size() : 0;
  if (numChildren != 0) {
      ...
      buf.append(numChildren);
      ...
      for (int i = 0; i < numChildren; i++) { ... }
  }
  ```